drivers/CMakeLists.txt

```diff
@@ -4,4 +4,6 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+add_subdirectory_ifdef(CONFIG_GPS_SIM gps_sim)
 add_subdirectory_ifdef(CONFIG_LTE_LINK_CONTROL lte_link_control)
+add_subdirectory_if_kconfig(sensor)
```

Comment:
nitpicking: both in `sensor_sim` and `gps_sim` `CMakeLists.txt `files you add the source files conditionally, which is redundant, since the directories are included conditionally already. We could simplify and keep the condition only for the top level `CMakeLists.txt` file.

Reply:
>Use add_subdirectory_ifdef to keep it consistent.

I see, was just replicating the way the sensor subfolder is added upstream here. I'll fix it to be consistent.

> nitpicking: both in sensor_sim and gps_sim CMakeLists.txt files you add the source files conditionally, which is redundant, since the directories are included conditionally already. We could simplify and keep the condition only for the top level CMakeLists.txt file.

Nitpicking is needed to keep things clean :) Will fix.

Reply:
You could see how I've done it here:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/426

Reply:
I skipped the `ifdef` part altogether, as it's already conditionally included with the same check in the `CMakeLists.txt` in the parent folder